defaults/main.yml

```diff
@@ -30,5 +30,14 @@ openio_netdata_oio_plugins:
     every: 60
     opts: "--fast"
 
+openio_netdata_oiofs_monitor: false
+openio_netdata_oiofs_endpoints: []
+# openio_netdata_oiofs_endpoints:
```

Comment:
Can you explain this default ? 
id/addr is not enough ?

Reply:
- id : the name of the endpoint 
- addr: the address to the stats socket
- full: collect all metrics (debug mode, useful for QA)
- enabled: controls the state of that collector